core/lib/constants/src/trusted_slots.rs

```diff
@@ -31,12 +31,24 @@ const INITIALIZER_INITIALING_SLOT: H256 = H256([
     0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
 ]);
 
+const ZEPPELINOS_ADMIN_SLOT: H256 = H256([
```

Comment:
ZEPPELINOS?

Reply:
Yes, I didn't find an ERC for that, but it's used here:
https://github.com/zeppelinos/zos-lib/blob/8a16ef3ad17ec7430e3a9d2b5e3f39b8204f8c8d/contracts/upgradeability/UpgradeabilityProxy.sol#L24
https://github.com/zeppelinos/zos-lib/blob/8a16ef3ad17ec7430e3a9d2b5e3f39b8204f8c8d/contracts/upgradeability/AdminUpgradeabilityProxy.sol#L26
and circle's FiatTokenProxy is almost the same. If you think another name is more suitable please suggest

Reply:
ZEPPELINOS is the legacy name for OPEN_ZEPPELIN library, I think we can change the name to OPEN_ZEPPELIN_ADMIN_SLOT

Reply:
Uh, I see, the slot is also legacy one